### PR TITLE
Cleaning up

### PR DESCRIPTION
**Pull request by aryans1204**

### CODE REVIEW

File: backend/src/models/user.js [hunk 1]
```diff
@@ -92,12 +99,13 @@ userSchema.statics.findByCredentials = async (email, password) => {
     const isMatch = await bcrypt.compare(password, user.password)
 
     if (!isMatch) {
-        throw new Error({error: "The password hashing is fucked"})
```
Comment (aryans1204) [inline backend/src/models/user.js:95]: whoops @dcxjn , this was for my personal debugging and I guess I mistakenly committed it to main 😮 you can change the error message to `either the password or email is incorrect`

Comment (dcxjn) [reply backend/src/models/user.js:95]: i've changed the error msg

**Comment by aryans1204** [comment]
Yup, checks have passed, it seems the issue was tracked in from my refactoring last night, but looks fixed. Is there anything more @dcxjn or can I merge this?

**Comment by dcxjn** [comment]
> Yup, checks have passed, it seems the issue was tracked in from my refactoring last night, but looks fixed. Is there anything more @dcxjn or can I merge this?

yep it can be merged, thanks!

**Pull request merged**